pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -296,11 +300,24 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 	}
 
 	var esLicenseType render.ElasticsearchLicenseType
+	var managerInternalTLSSecret *corev1.Secret
 	if managementClusterConnection == nil {
 		if esLicenseType, err = utils.GetElasticLicenseType(ctx, r.client, reqLogger); err != nil {
 			r.status.SetDegraded("Failed to get Elasticsearch license", err.Error())
 			return reconcile.Result{}, err
 		}
+
+		managerInternalTLSSecret, err = utils.ValidateCertPair(r.client,
```

Comment:
Depending on which pull is merged first, you may wanna change the key to "", such that it is not checked.
https://github.com/tigera/operator/pull/1260/